lndr-backend/src/Lndr/Types.hs

```diff
@@ -127,6 +128,11 @@ data BilateralCreditRecord = BilateralCreditRecord { creditRecord :: CreditRecor
                                                    } deriving (Show, Generic)
 $(deriveJSON (defaultOptions { omitNothingFields = True }) ''BilateralCreditRecord)
 
+data SettlementVerification = SettlementVerification { creditorAddr     :: Address
```

Comment:
I don't think you should create a new type for this. Additionally, it doesn't make sense for `txnHash` to be `Maybe TransactionHash` since  to verify a settlement, you'd absolutely need a tx hash.

Let's stick to `txHash` in the code base (as opposed to `txnHash`) in order to stay consistent.

Reply:
(rule of thumb: if you need a small record type only once in your code, it probably shouldn't exist)